### PR TITLE
Make hydra generator create wrapper config files

### LINKED ISSUE #339: Update generator to create default Fcrepo and Solr wrapper config files
**STORY**
As a developer, I would like the hydra generator to provide config files for fcrepo_wrapper and solr_wrapper, so I can easily start those services for development and test environments with hydra-friendly settings.

**DETAILS**
We need a separate config file for development and test for solr_wrapper and fcrepo_wrapper - so four files total:

```
#config/solr_wrapper_development.yml
# version: 6.0.0
port: 8983
instance_dir: tmp/solr-development
collection:
    persist: true
    dir: solr/config
    name: hydra-development
```

```
#config/solr_wrapper_test.yml
# version: 6.0.0
port: 8985
instance_dir: tmp/solr-test
collection:
    persist: false
    dir: solr/config
    name: hydra-test
```

```
#config/fcrepo_wrapper_development.yml
port: 8984
enable_jms: false
fcrepo_home_dir: tmp/fcrepo4-development-data
```

```
#config/fcrepo_wrapper_test.yml.sample
port: 8986
enable_jms: false
fcrepo_home_dir: tmp/fcrepo4-test-data
```

**ACCEPTANCE**
- [ ] After I run the hydra generator, there is a solr_wrapper_test.yml generated in my ./config directory
- [ ] After I run the hydra generator, there is a solr_wrapper_development.yml generated in my ./config directory
- [ ] After I run the hydra generator, there is a fcrepo_wrapper_test.yml generated in my ./config directory
- [ ] After I run the hydra generator, there is a fcrepo_wrapper_development.yml generated in my ./config directory

### PR DESCRIPTION
**Pull request by mark-dce**
Fixes #339

### CODE REVIEW

File: hydra-core/lib/generators/hydra/head_generator.rb [hunk 1]
```diff
@@ -111,5 +111,13 @@ def inject_hydra_user_behavior
         puts "     \e[31mFailure\e[0m  Hydra requires a user object in order to apply access controls. This generators assumes that the model is defined in the file #{file_path}, which does not exist.  If you used a different name, please re-run the generator and provide that name as an argument. Such as \b  rails -g hydra:head client"
       end
     end
+    
+    def create_wrapper_configuration
```
Comment (cbeer) [inline hydra-core/lib/generators/hydra/head_generator.rb]: Should we move this to a new generator (and still call it here), such that people who've already generated hydra can easily run this without other side-effects?

File: hydra-core/lib/generators/hydra/templates/config/fcrepo_wrapper_development.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#config/fcrepo_wrapper_development.yml
+port: 8984
+enable_jms: false
+fcrepo_home_dir: tmp/fcrepo4-development-data
```
Comment (cbeer) [inline hydra-core/lib/generators/hydra/templates/config/fcrepo_wrapper_development.yml]: If we want to be nice to downstream users, can we add a trailing new line to these yml files?

File: hydra-core/spec/lib/hydra_generator_notspec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
```
Comment (cbeer) [inline hydra-core/spec/lib/hydra_generator_notspec.rb]: Can you say something about why this is a "`notspec`"? are we just trying to ignore it?

**Comment by mark-dce** [comment]
Moved code to active_fedora as requested in #339

**Comment by cbeer** [comment]
👍

**Pull request closed without merging**